readme.md

```diff
@@ -12,7 +12,7 @@ PRIVACY: TEZOS REWARD DISTRIBUTOR COLLECTS ANONYMOUS STATISTICS. PLEASE READ OUR
 
 TRD is a software for distributing staking rewards of Tezos delegators, introduced in detail in this [Medium article](https://medium.com/@huseyinabanox/tezos-reward-distributor-e6588c4d27e7). This is not a script but a full scale application which can continuously run in the background as a Linux service. It can track cycles and make payments. However, it does not have to be used as a service, but it can also be used interactively.
 
-The documentation can be found [here](https://tezos-reward-distributor-organization.github.io/tezos-reward-distributor/). You can ask for support on our [Slack channel](https://tezos-baking.slack.com/messages/CQ35AM8KE) or [file an issue](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/issues).
+The documentation can be found [here](https://tezos-reward-distributor-organization.github.io/tezos-reward-distributor/). You can ask for support on our [Slack channel](https://join.slack.com/t/tezos-baking/shared_invite/zt-yqxeszcy-rpvYmBtXr5oewh6M0DFkyQ) or [file an issue](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/issues).
```

Comment:
Suggested text for slack part.
You can ask for support on the TRD channel of the tezos-baking slack group, if you are a member of this group you can view the channel here [TRD Slack Channel](https://tezos-baking.slack.com/messages/CQ35AM8KE),  if you are not a member you can join the group and create an account here [Tezos-baking Slack group](https://join.slack.com/t/tezos-baking/shared_invite/zt-yqxeszcy-rpvYmBtXr5oewh6M0DFkyQ) then find the trd channel from the channel list.

Reply:
The link to the channel should still be left in place, we need just to add the joining part. Some people will be members already.